src/states/Game.gd

```diff
@@ -72,21 +76,30 @@ func server_disconnected() -> void:
 	Global.menu_to_load = "main_menu"
 	get_tree().change_scene("res://src/states/Menu.tscn")
 
+	
+func get_targets() -> Array:
```

Comment:
Add a comment explaining why this is her. Namely that we want to get all the targets that are not going to be despawned next frame.